pkg/visor/visorconfig/services.go

```diff
@@ -15,9 +15,9 @@ import (
 // Fetch fetches the service URLs & ip:ports from the config service endpoint
 func Fetch(mLog *logging.MasterLogger, serviceConfURL string, stdout bool) (services *Services) {
 
-	serviceConf := fmt.Sprint("http://", serviceConfURL)
+	serviceConf := fmt.Sprint(serviceConfURL)
 	client := http.Client{
-		Timeout: time.Second * 2, // Timeout after 2 seconds
+		Timeout: time.Second * 30, // Timeout after 30 seconds
```

Comment:
30 seconds is way too long. If the request does not finish within 5 seconds, we can assume there is an error and let the application go forward.

Reply:
This is a critical step, especially for any custom deployment setting. I want at least 10-15 seconds there. I know its not a lot of data but we have had bad issues before with too short of timeouts, so it should have as much time as it needs to get a good response right there if it is indeed possible.

For most people it won't matter; but for the edge case of a high latency, low bandwidth connection, 30 seconds should be enough.

I suppose this should fail explicitly on inability to fetch from a custom service conf URL.